src/download_data.py

```diff
@@ -109,18 +109,11 @@ def download_pudl_data(zenodo_url: str):
 def download_pudl(zenodo_url, pudl_version):
     r = requests.get(zenodo_url, params={"download": "1"}, stream=True)
     # specify parameters for progress bar
-    total_size_in_bytes = int(r.headers.get("content-length", 0))
     block_size = 1024 * 1024 * 10  # 10 MB
-    downloaded = 0
+    logger.info("    Downloading PUDL data...")
     with open(downloads_folder("pudl.tgz"), "wb") as fd:
         for chunk in r.iter_content(chunk_size=block_size):
-            print(
-                f"    Downloading PUDL. Progress: {(round(downloaded/total_size_in_bytes*100,2))}%   \r",
-                end="",
-            )
```

Comment:
We may want to keep the PUDL download progress update here, just because the download takes so long. It's nice to know that something is still happening and that your computer isn't frozen or something.

Reply:
We'd have to keep this as a print statement though, rather than a logging statement, which would be the only print statement in the entire pipeline. Perhaps we can just add a message to the logging statement saying that the file is large and download takes a long time?

Reply:
I don't think having a `print` statement is a problem. It'll show up for developers watching their terminal, but not in the logs (which is totally fine since it's just a progress update).

Reply:
Ok added this back in